lib/aws/rails/notifications_instrument_plugin.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'aws-sdk-core'
+
+module Aws
+  module Rails
+    # @api private
+    class NotificationsInstrument < Seahorse::Client::Plugin
```

Comment:
I'm not a fan of the name.. how about just Notifications (already under Aws::Rails)

Reply:
Wanted to differentiate between pub and sub here, but agree the name is long and awkward.